skimage/__init__.py

```diff
@@ -56,18 +56,21 @@
 
 """
 
-import os.path as _osp
-import imp as _imp
-import functools as _functools
-import warnings as _warnings
+import os.path as osp
+import imp
+import functools
+import warnings
```

Comment:
Notice the `del` at the bottom, they are cleared there. 



—
Sent from Mailbox

On Sun, Jul 12, 2015 at 11:01 AM, Josh Warner <notifications@github.com>
wrote:

>> @@ -56,18 +56,21 @@
>>  
>>  """
>>  
>> -import os.path as _osp
>> -import imp as _imp
>> -import functools as _functools
>> -import warnings as _warnings
>> +import os.path as osp
>> +import imp
>> +import functools
>> +import warnings
> I think these were aliased previously to hide them from the base package namespace.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/scikit-image/scikit-image/pull/1599/files#r34422930